optuna/samplers/_tpe/sampler.py

```diff
@@ -293,14 +294,17 @@ def __init__(
             dict[str, Callable[[CategoricalChoiceType, CategoricalChoiceType], float]] | None
         ) = None,
     ) -> None:
-        if consider_prior is False:
+        if not consider_prior:
             msg = _deprecated._DEPRECATION_WARNING_TEMPLATE.format(
                 name="`consider_prior`", d_ver="4.3.0", r_ver="6.0.0"
             )
-            warnings.warn(msg, FutureWarning)
+            warnings.warn(
+                f"{msg} From v4.3.0 onward, `consider_prior` automatically falls back to `True`.",
+                FutureWarning,
+            )
 
         self._parzen_estimator_parameters = _ParzenEstimatorParameters(
-            consider_prior,
+            True,
```

Comment:
Could you use the keyword args here for more clarity?

```suggestion
            consider_prior=True,
```